spring-cloud-modules/spring-cloud-aws-v3/src/test/java/com/baeldung/spring/cloud/aws/sqs/SpringCloudAwsSQSLiveTest.java

```diff
@@ -0,0 +1,84 @@
+package com.baeldung.spring.cloud.aws.sqs;
+
+import static com.baeldung.spring.cloud.aws.sqs.UserEventListeners.EVENT_TYPE_CUSTOM_HEADER;
+import static org.awaitility.Awaitility.await;
+
+import java.time.Duration;
+import java.util.Map;
+import java.util.UUID;
+
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import io.awspring.cloud.sqs.operations.SqsTemplate;
+
+public class SpringCloudAwsSQSLiveTest extends BaseSqsIntegrationTest {
+
+    private static final Logger logger = LoggerFactory.getLogger(SpringCloudAwsSQSLiveTest.class);
+
+    @Autowired
+    private SqsTemplate sqsTemplate;
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private EventQueuesProperties eventQueuesProperties;
+
+    @Test
+    void givenAStringPayload_whenSend_shouldReceive() {
+        // given
+        var userName = "Albert";
+
+        // when
+        sqsTemplate.send(to -> to.queue(eventQueuesProperties.getUserCreatedByNameQueue())
+            .payload(userName));
+        logger.info("Message sent with payload {}", userName);
+
+        // then
+        await().atMost(Duration.ofSeconds(3))
+            .until(() -> userRepository.findByName(userName)
+                .isPresent());
+    }
+
+    @Test
+    void givenARecordPayload_whenSend_shouldReceive() {
+        // given
+        String userId = UUID.randomUUID()
+            .toString();
+        var payload = new UserCreatedEvent(userId, "John", "john@baeldung.com");
+
+        // when
+        sqsTemplate.send(to -> to.queue(eventQueuesProperties.getUserCreatedFifoQueue())
```

Comment:
This test does not seem to do anything FIFO specific, but I'm guessing the point was just to show "it's possible to use a FIFO queue" in one of the tests

Reply:
Yeah, that was a last minute addition to this PR but I think it's worth mentioning it.

I plan to explain some basic things about what gets configured automatically for FIFO queues, such as acknowledgement.